src/N3StreamParser.js

```diff
@@ -18,12 +18,13 @@ export default class N3StreamParser extends Transform {
         case 'end':   onEnd = callback; break;
         }
       },
-    },
+    }, {
       // Handle quads by pushing them down the pipeline
-      (error, quad) => { error && this.emit('error', error) || quad && this.push(quad); },
+      onQuad: (error, quad) => { error && this.emit('error', error) || quad && this.push(quad); },
       // Emit prefixes through the `prefix` event
-      (prefix, uri) => { this.emit('prefix', prefix, uri); },
-    );
+      onPrefix: (prefix, uri) => { this.emit('prefix', prefix, uri); },
+      onComment: comment => { this.emit('comment', comment); },
```

Comment:
This one is unconditional though, so there will be performance implications. (That's where a constructor param would perhaps help.)

Reply:
This will now not happen by default anymore: a constructor parameter is required now